static/js/service-worker-registration.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-env browser */
+(function() {
+  'use strict';
+
+  // Check to make sure service workers are supported in the current browser,
```

Comment:
`chromestatus.com` forcibly redirects to HTTPS, so there shouldn't be a need to implement this check.